PanelKit/PanelManager/PanelManager.swift

```diff
@@ -79,25 +79,11 @@ public extension PanelManager where Self: UIViewController {
 extension PanelManager {
 	
 	var panelPinnedLeft: PanelViewController? {
-		
-		for panel in panels {
-			if panel.pinnedSide == .left {
-				return panel
-			}
-		}
-		
-		return nil
-	}
+        return panels.filter{ $0.pinnedSide == .left }.first
```

Comment:
Nitpicking here, but I'm pretty sure putting a space between `filter` and `{` is commonplace